code/modules/vehicles/sealed.dm

```diff
@@ -1,5 +1,6 @@
 /obj/vehicle/sealed
 	var/enter_delay = 20
+	var/mouse_pointer = null
```

Comment:
remoof null

Reply:
>living in 2018 and still doing this

qusty wyd

Reply:
i think i copy pasted the line from mecha code.

tbf i shouldve known going into mecha code

Reply:
>living in 2018 and still not following code standards

Reply:
reeeeee

Reply:
atleast you cant get into the clowncar without having the key >:(